Turbolist3r not working

mijanurrahman opened the pull request:
![turbo](https://user-images.githubusercontent.com/5920847/37715453-43a47d92-2d46-11e8-92cf-657f3cc67727.jpg)

fleetcaptain committed:
Update README.md

fleetcaptain committed:
Update README.md

fleetcaptain committed:
Update README.md

Added screenshots, updated Credits

fleetcaptain committed:
Delete sublist3r.py

fleetcaptain committed:
Add files via upload

fleetcaptain committed:
Update README.md

fleetcaptain committed:
Update turbolist3r.py

Changed the analysis DNS requests to use dnslib for better subdomain takeover detection. The dns.resolver library could miss out on some subdomains pointing to vulnerable cloud services that returned NXDOMAIN status since the cloud service did not exist but the target domain's pointer record still existed and pointed to it.

fleetcaptain committed:
Update turbolist3r.py

Edited analysis section to be more resilient against errors. Found subdomain candidates that began with a dot (ex .domain.com) caused the DNS query to choke. Modified exception catch block to catch unknown errors and continue around to try the next subdomain candidate.

kartikeyap committed:
Added dnslib as a requirement in requirements.txt

codingo commented:
This is for a fork of this project (https://github.com/fleetcaptain/Turbolist3r) and not this project itself. This issue should be closed.

fleetcaptain committed:
Merge pull request #1 from kartikeyap/master

Added dnslib as a requirement in requirements.txt

kartikeyap committed:
Uncommented import for subbrute module

fleetcaptain committed:
Merge pull request #2 from kartikeyap/master

Uncommented import for subbrute module

ZeroDot1 commented:
$ python turbolist3r.py -v -d coinhive.com
  File "turbolist3r.py", line 1025
    subdomain = subdomain.replace("From http://PTRarchive.com: ", "")
                                                                    ^
TabError: inconsistent use of tabs and spaces in indentation

Commit:
Multiple updates:
- Added debug function (--debug) to print verbose DNS information
- Fixed a bug where CNAME records could be incorrectly reported as A records

Commit:
- Added a debugging option (--debug) that prints verbose DNS information as it is processed during the analysis module (-a)
- Fixed a bug where CNAME records could be incorrectly reported as A records

fleetcaptain committed:
Update README.md

Fixed formatting issue

Commit:
Fixed different python3 bugs:
- wrong intendation
- mix of "print ..." and "print(...)"
- minor improvents (https, headers as static var)

Commit:
- Added cloudflare resolvers
- Fixed deadlock based on https://github.com/persebek/Sublist3r/commit/75144adaa62afb06ea3448e1212019bcbdcb1db9

fleetcaptain committed:
Merge pull request #3 from Damian89/master

Major bugfixes

Commit:
Changes:
- The -a option no longer requires an argument and does not save RDNS analysis by default
- Use --saverdns <file> to save analysis output
- Added capability to perform RDNS on subdomains loaded from file (--inputfile)
- Other minor edits

fleetcaptain committed:
Update README.md

Added help for new options

fleetcaptain committed:
Update README.md

Update first screenshot to reflect correct option usage

Commit:
Remove resolver 75.75.75.75, kept getting errors
Error messages now print which resolver was involved in the error

fleetcaptain committed:
Update turbolist3r.py

Server 75.75.75.75 seems prone to frequent timeouts and errors, removing from pool of resolvers

Commit:
Merge branch 'master' of https://github.com/fleetcaptain/Turbolist3r

Commit:
Converted indentation from spaces to tabs universally across the whole script

Commit:
Fix SHA hashing and Unicode issue in python3

touhidshaikh committed:
Record Lines Colour Changed 

Record Lines Colour Changed (CNAME and A)

touhidshaikh committed:
Error Line in Red

Error Line in Red

fleetcaptain committed:
Merge pull request #4 from touhidshaikh/master

Record Line and Error line Colour Changed

touhidshaikh committed:
Added Table View for records display.

Added Table View for records display. A separate table for CNAME and A records.

touhidshaikh committed:
Update turbolist3r.py

touhidshaikh committed:
Table Align to Left.

Aligned Table to left for better looks. ;)

fleetcaptain committed:
Merge pull request #5 from touhidshaikh/touhidshaikh-table-added

Thanks for the contribution!

fleetcaptain committed:
Revert "Table added"

fleetcaptain committed:
Merge pull request #6 from fleetcaptain/revert-5-touhidshaikh-table-added

Revert "Table added"

Commit:
New feature: DNS servers/resolvers can now be loaded from a file. Use the command line option -r/--resolvers to specify the path to the resolvers file.
Enter one DNS server IP address per line.

fleetcaptain committed:
Update README.md

fleetcaptain committed:
Update README.md

Commit:
Uncommented out the subbrute import statement that was preventing bruteforce from running

Commit:
Merge branch 'master' of https://github.com/fleetcaptain/Turbolist3r

fleetcaptain committed:
Update README.md

Add clarification about subbrute

gavin-anders committed:
Support for Python3

fleetcaptain committed:
Merge pull request #8 from gavin-anders/patch-1

Correct print when using python3

Commit:
Switched use of dns.resolver to dnspython

fleetcaptain committed:
Update requirements.txt

dnspython no longer needed

fleetcaptain committed:
Update README.md

Update README to reflect only dnslib is needed now

Commit:
Merge branch 'dev'

Github didn't like web PR so following manual instructions

Commit:
Merge branch 'master' of https://github.com/fleetcaptain/Turbolist3r

fixing conflicts

fleetcaptain committed:
Update README.md

fleetcaptain committed:
Update turbolist3r.py

fix merge error

vitezslav-lindovsky committed:
Add -q --quiet switch to show only result

fleetcaptain committed:
Merge pull request #10 from vitezslav-lindovsky/add-quiet-turbolister

Add -q --quiet switch to show only result

Commit:
- Updated README to reference new -q option
- Added check to a few more output statements to respect silent option

Commit:
Parsing update for dnsdumpster.com

fleetcaptain committed:
Merge pull request #11 from 5hizzle/fix/dnsdumpster-parse

Parsing update for dnsdumpster.com

fleetcaptain committed:
Some subdomains were not getting scraped correctly from data source. Added logic to clean these up

wopperer committed:
Fix issue https://github.com/aboul3la/Sublist3r/issues/236 to make it work with Python 3.9.5

fleetcaptain committed:
Merge pull request #12 from wopperer/fix_for_python_3.9.5

Fix issue https://github.com/aboul3la/Sublist3r/issues/236 to make it…

fleetcaptain committed:
Update README.md

usmankhan7327 committed:
Fix DNSDumpster CSRF token parsing to prevent IndexError

fleetcaptain committed:
Merge pull request #15 from usmankhan7327/fix-dnsdumpster-token

Fix DNSDumpster CSRF token IndexError

usmankhan7327 committed:
Fix all invalid escape sequence warnings across regex and banner strings

fleetcaptain committed:
Merge pull request #16 from usmankhan7327/fix-invalid-escape-sequences

Fix all invalid escape sequence warnings across regex and banner strings
